manifests/0000_07_cluster-network-operator_01_crd.yaml

```diff
@@ -67,14 +67,27 @@ spec:
               type: array
               items:
                 type: object
-                required: ["type", "name", "rawCNIConfig"]
+                required: ["type", "name"]
                 properties:
                   type:
                     type: string
                   name:
                     type: string
                   rawCNIConfig:
                     type: string
+                  macVlanConfig:
+                    type: object
+                    required: ["master" ]
+                    properties:
+                      master:
+                        type: string
+                      ipam:
+                        type: string
+                      mode:
+                        type: string
+                      mtu:
```

Comment:
when would this not just be the mtu of the master?

Reply:
To optimize TCP traffic (to avoid segmentation) for specific target, but it is of course not usual stuff.

Reply:
@s1061123 do we have a concrete use-case for having the MTU at this point? eg do we know of specific situations or scenarios that the MTU of the master wouldn't work?

Reply:
removed mtu.

Reply:
I don't have clear use cases with detailed situation, so removed.